app/code/Magento/Checkout/Test/Mftf/Section/CheckoutSuccessMainSection.xml

```diff
@@ -16,7 +16,7 @@
         <element name="orderLink" type="text" selector="a[href*=order_id].order-number" timeout="30"/>
         <element name="orderNumberText" type="text" selector=".checkout-success > p:nth-child(1)"/>
         <element name="continueShoppingButton" type="button" selector=".action.primary.continue" timeout="30"/>
-        <element name="createAnAccount" type="button" selector="input[value='Create an Account']" timeout="30"/>
+        <element name="createAnAccount" type="button" selector="#registration a.action.primary span[data-bind*=&quot;i18n: 'Create an Account'&quot;]" timeout="30"/>
```

Comment:
@p-bystritsky please modify all selectors to satisfy https://devdocs.magento.com/guides/v2.3/coding-standards/code-standard-demarcation.html

Reply:
@orlangur done.

Reply:
@p-bystritsky I don't see... In functional tests data attributes, names etc must be used. No IDs, CSS classes and such allowed.

Reply:
@orlangur updated.